cmd/minikube/cmd/start.go

```diff
@@ -182,5 +187,8 @@ func init() {
 	startCmd.Flags().StringSliceVar(&insecureRegistry, "insecure-registry", nil, "Insecure Docker registries to pass to the Docker daemon")
 	startCmd.Flags().StringSliceVar(&registryMirror, "registry-mirror", nil, "Registry mirrors to pass to the Docker daemon")
 
+	startCmd.Flags().BoolVarP(&startDockerRegistry, "internal-registry", "", false, "Should the OpenShift internal Docker registry be started?")
```

Comment:
Just registry will do for the flag I think. Alternatively, deploy-registry & deploy-router for consistency?